Spanner/tests/Snippet/SpannerClientTest.php

```diff
@@ -255,6 +255,7 @@ public function factoriesProvider()
             [Int64::class, 'int64'],
             [Duration::class, 'duration'],
             [CommitTimestamp::class, 'commitTimestamp'],
+            [Numeric::class, 'numeric'],
```

Comment:
Add `use Google\Cloud\Spanner\Numeric` to the top of this file to fix the test failure.